.expeditor/scripts/prep_and_run_tests.ps1

```diff
@@ -3,7 +3,8 @@ param(
     [Parameter(Position=0)][String]$TestType
 )
 
-. { Invoke-WebRequest -useb https://omnitruck.chef.io/chef/install.ps1 } | Invoke-Expression; install -channel "current" -project "chef-foundation" -v $CHEF_FOUNDATION_VERSION
+. { Invoke-WebRequest -useb https://omnitruck.chef.io/chef/install.ps1 } | Invoke-Expression; install -channel "current" -project "chef-foundation" -v ${env:CHEF_FOUNDATION_VERSION}
```

Comment:
You don't need the braces for that to evaluate. Correct syntax is `$env:CHEF_FOUNDATION_VERSION`. If you have a thing that needs to evaluate a thing first and drop the answer inline then you would do this: `$($env:CHEF_FOUNDATION_VERSION)`

Reply:
Ah, looks like it was just a carryover from `.expeditor/scripts/omnibus_chef_build.ps1`... changing and making the other consistent.